dev/com.ibm.ws.openapi.ui/swagger-ui/dist/index.html

```diff
@@ -5,7 +5,7 @@
     <title>Swagger UI</title>
     <link rel="icon" type="image/png" href="./favicon-32x32.png" sizes="32x32" />
     <link rel="icon" type="image/png" href="./favicon-16x16.png" sizes="16x16" />
-    <script defer src="liberty-swagger-ui-bundle-f4a5a5cf1b006a74f119.js"></script><link href="liberty-swagger-ui-a44cef3790da8bb191bd.css" rel="stylesheet">
+    <script defer src="liberty-swagger-ui-bundle-f46b8f492bc9c6c9d41c.js"></script><link href="liberty-swagger-ui-a44cef3790da8bb191bd.css" rel="stylesheet">
```

Comment:
It looks like you've correctly rebuilt the UI after making your change, but we will need to run the manual regression test: https://github.com/OpenLiberty/openapi-ui-test-app